mongodb/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM gitpod/workspace-full:latest

```

No need to specify `:latest`, it's the default value.

```suggestion
FROM gitpod/workspace-full
```